### ISSUE TITLE
can not compile the CRNN

### ISSUE THREAD

**Issue by AlwaysTrying** [MhLiao/TextBoxes#101]
Hello, 
I tried many solutions and check so many times the installation of the libraries, I don't know what I am missing but I couldn't solve this problem from few days, 

(my_env) +++@***:~/TextBoxes_plusplus/crnn/src$ sh build_cpp.sh 
-- The C compiler identification is GNU 5.4.0
-- The CXX compiler identification is GNU 5.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found OpenMP_C: -fopenmp (found version "4.0") 
-- Found OpenMP_CXX: -fopenmp (found version "4.0") 
-- Found OpenMP: TRUE (found version "4.0")  
-- Configuring done
-- Generating done
-- Build files have been written to: /home/+++/TextBoxes_plusplus/crnn/src/cpp/build
Scanning dependencies of target crnn
[ 25%] Building CXX object CMakeFiles/crnn.dir/init.cpp.o
/home/+++/TextBoxes_plusplus/crnn/src/cpp/init.cpp:1:19: fatal error: lua.hpp: No such file or directory
compilation terminated.
CMakeFiles/crnn.dir/build.make:62: recipe for target 'CMakeFiles/crnn.dir/init.cpp.o' failed
make[2]: *** [CMakeFiles/crnn.dir/init.cpp.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/crnn.dir/all' failed
make[1]: *** [CMakeFiles/crnn.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
cp: cannot stat '*.so': No such file or directory

however, lua is installed 
$ lua -v
Lua 5.2.3  Copyright (C) 1994-2013 Lua.org, PUC-Rio

and the files are present here ;
/lua-5.3.5/src$ ls
lapi.c  ...........  lutf8lib.c  Makefile  lauxlib.h   lcode.o     ldebug.c  lfunc.c  linit.o   lmathlib.o  lobject.o   lparser.o  lstrlib.c  ltm.h      lua.h      lutf8lib.o  lauxlib.o   lcorolib.c  ldebug.h  lfunc.h  liolib.c  lmem.c      lopcodes.c  lprefix.h  lstrlib.o  ltm.o      lua.hpp    lvm.c  lbaselib.c  lcorolib.o  ldebug.o  lfunc.o  liolib.o  lmem.h      lopcodes.h  lstate.c   ltable.c   lua        lualib.h   lvm.h lbaselib.o  lctype.c    ldo.c     lgc.c    llex.c    lmem.o      lopcodes.o  lstate.h   ltable.h   luac       lua.o      lvm.o

any verification i should do  to find the source of the problem plz ?
best,

**Comment by AlwaysTrying** [comment]
hello @MhLiao  , could you help me with this issue please, i tried many solutions proposed in the issues but it does not work and i got new errors every time...

**Comment by MhLiao** [comment]
@AlwaysTrying Did you try the docker?

**Comment by AlwaysTrying** [comment]
no, what do you mean ?

**Comment by MhLiao** [comment]
There is a [docker file](https://github.com/bgshih/crnn/blob/master/Dockerfile) for CRNN. Otherwise, there is also a docker in this [rep](https://github.com/MhLiao/TextBoxes_plusplus).

**Comment by AlwaysTrying** [comment]
Yes i saw this , but I can not use it, I am working on the server and it is not installed ...
I work on virtual machine . Can I run it on virtual machine plz?
